.github/workflows/ci-llama.yaml

```diff
@@ -0,0 +1,70 @@
+name: Integration Tests
```

Comment:
You can change the name here because we already have an Integration Tests. Maybe something more specific to llama?